### DISCUSSION TITLE
Any way to setup the custom button into an music source finder?

### DISCUSSION THREAD

**Discussion by Elasder123** [sll552/DiscordBee#202] [Q&A] [answered]
So I want the Custom button to show the song info online. Like when you click it, it will bring up a youtube or Spotify tab on your browser 
is that possible?

**Comment by sll552** [comment, marked as answer]
You can use the placeholders in the url, so if whatever service you want to open has a search url then something like that should be possible. For YT for example it could look something like `https://www.youtube.com/results?search_query=[Artist]-[TrackTitle]`

**Reply by Elasder123** [reply]
Thanks for telling me that! I will try it then!
Edit: I tried it and it does indeed work! But instead of Using the tag "Artist" I used instead "Album" so it searches for the album instead which works better for me :) Just a personal preference 
2nd Edit: Found out that you can have multiple tags. Example [Album]-[Artist]-[TrackTitle]
